mainwindow.py

```diff
@@ -169,6 +171,20 @@ def openservice10(self):
         else:
             self.update_status(f"10 Button clicked. But unable to open the service window as CAN is not connected")
         return
+
+    def openservice11(self):
+        if(Is_CanConnected == True):
+            self.window11 = QMainWindow()
+            self.ui11 = dsc.Ui_Service11()
```

Comment:
this the class Ui_Service11 is not present in dsc module but er module. so update it as er. instead of dsc.

Reply:
Thank you for pointing this out sir . I had actually noticed that the Ui_Service11 class belongs to the er module and had updated it in my local environment on VS Code. However, I forgot to push the changes to GitHub. I also realize that I made a few unnecessary commits today . I'll correct the mistake and ensure the correct changes are pushed.